s5_base/run.sh

```diff
@@ -70,12 +67,12 @@ if [ $stage -le 4 ]; then
   cat data/train/text | cut -d' ' -f2- | sed -e \
  's/[,.?!]//g' > data/text_lm_training.txt
 
-  mkdir -p data/lang_3gram
+  mkdir -p data/lang_3gsmall
 
   # language  modeling files are typically in data/lang_Xgram
   $big_memory_cmd logs/make_LM_3gsmall.log local/make_LM.sh --order 3 \
   --small true --carpa false data/text_lm_training.txt data/lang/ \
-  data/local/dict/lexicon.txt data/lang_3gram
+  data/local/dict/lexicon.txt data/lang_3gsmall
```

Comment:
I will change this part. And isn't it then simplest if we call it just lang_3g? Or combine lang, lang_3g and lang_4g into one dir?

Reply:
i could have a general lang directory where the rest are nested underneath. How about that?

Reply:
I was thinking since everything is the same in these directories other than G.fst and G.carpa that the small one would be G.fst and the large one G.carpa like Robert did for the dialect data.I can create the code that would do that